setup.rst

```diff
@@ -225,26 +225,15 @@ require --no-unpack ...`` option to disable unpacking.
 Checking Security Vulnerabilities
 ---------------------------------
 
-The ``symfony`` binary created when you `install Symfony CLI`_ provides a command to
-check whether your project's dependencies contain any known security
-vulnerability:
+A good and recommended security practice is to check from time to time whether your
+project's dependencies contain any known security vulnerability. You can leverage the
+`Local PHP Security Checker`_ to do so.
 
-.. code-block:: terminal
-
-    $ symfony check:security
-
-A good security practice is to execute this command regularly to be able to
+You can also execute this process regularly to be able to
 update or replace compromised dependencies as soon as possible. The security
 check is done locally by fetching the public `PHP security advisories database`_,
 so your ``composer.lock`` file is not sent on the network.
 
-.. tip::
```

Comment:
This tip is also valid for the security checker, so it should be kept IMHO.

Reply:
added back